docs/src/docs/asciidoc/gradle-plugin.adoc

```diff
@@ -227,6 +227,38 @@ Currently, this feature requires the execution of the tests in the classic "JVM"
 
 NOTE: This plugin requires JUnit Platform 1.8 or higher.
 
+[[mockito-bytebuddy-support]]
+=== Mockito / Byte Buddy support
+
+Mockito is supported starting from version 4.5.0 (ByteBuddy >= 1.12.9) with GraalVM >= 22.1.
+For the time being, a bit of additional configuration is needed to utilize it:
```

Comment:
Is it really for the time being? I understood future releases of GraalVM/BB will support OOTB. I would rather rephrase it:

> However, you need to add the following configuration to make it work with GraalVM 22.1: